src/Components/App/App.js

```diff
@@ -4,7 +4,9 @@ import Header from '../Header/Header';
 import { Form } from '../Form/Form';
 import React, {Component} from 'react';
 import {Route} from 'react-router-dom';
-
+import { getAllMovies, getAllPodcasts, getAllCardGames, getAllMusic, getAllBoardGames} from '../../apiCalls.js'
```

Comment:
I am ok with the idea about app handling this API calls

Reply:
Yeah, I think I agree with this.  It will be easiest to access this data across all components if it exists as part of App's state.